lib/mws/orders/client.rb

```diff
@@ -5,7 +5,8 @@ module Orders
     # With the MWS Orders API, you can list orders created or updated during a
     # time frame you specify or retrieve information about specific orders.
     class Client < ::Peddler::Client
-      path '/Orders/2013-09-01'
+      version '2013-09-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.